generic3g/vertical/ModelVerticalGrid.F90

```diff
@@ -28,9 +30,9 @@ module mapl3g_ModelVerticalGrid
       integer :: num_levels = -1
       type(StringVector) :: variants
 
-!#      character(:), allocatable :: short_name
-!#      character(:), allocatable :: standard_name
-!#      type(ESMF_Field) :: reference_field
+      ! character(:), allocatable :: short_name
+      ! character(:), allocatable :: standard_name
+      ! type(ESMF_Field) :: reference_field
```

Comment:
Do not modify comments starting `!#` except to fully delete them.  They are used by my editor to comment/uncomment sections.       Either delete the comment completely, or leave it as it is.
```suggestion
      !# character(:), allocatable :: short_name
      !# character(:), allocatable :: standard_name
      !# type(ESMF_Field) :: reference_field
```

Reply:
Interesting! My editor uses the standard `!` for commenting/un-commenting sections. I'll put the `#` back, but keep the indentation.